docs/development/getting-started.rst

```diff
@@ -231,7 +243,7 @@ https://github.com/chadoe/docker-cleanup-volumes)
 Building Styles
 ---------------
 
-Styles are written in the scss variant of Sass and compiled using Gulp. They
+Styles are written in the scss variant of Sass and compiled using ``gulp``. They
```

Comment:
You've got a linting error on this line, it's too long.

Reply:
Thanks! I thought `make docs` would catch that; should have run `make lint`. Fixed.